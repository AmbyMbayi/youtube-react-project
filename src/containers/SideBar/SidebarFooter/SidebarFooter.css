.footer__block {
    color: rgb(219, 218, 218);
    padding-bottom: 10px;
    padding-left: 20px;
    padding-top: 10px;
    margin-bottom: 10px;
}