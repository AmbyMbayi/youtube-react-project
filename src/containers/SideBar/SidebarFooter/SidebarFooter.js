import React from 'react';
import './SidebarFooter.css';

function SidebarFooter() {
	return (
		<div className="footer__block">
			<div>A(c) TealTube using React</div>
		</div>
	);
}

export default SidebarFooter;
