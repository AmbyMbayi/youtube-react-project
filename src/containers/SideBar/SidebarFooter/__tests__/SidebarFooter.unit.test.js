import React from 'react';
import { shallow } from 'enzyme';
import SidebarFooter from '../SidebarFooter';

test('renders SidebarFooter', () => {
	const wrapper = shallow(<SidebarFooter />);
	expect(wrapper).toMatchSnapshot();
});
