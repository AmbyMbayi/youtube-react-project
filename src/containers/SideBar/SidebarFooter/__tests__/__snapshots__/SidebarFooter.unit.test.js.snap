// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders SidebarFooter 1`] = `
<div
  className="footer__block"
>
  <div>
    A(c) TealTube using React
  </div>
</div>
`;
